archive/v1/README.md

```
@@ -0,0 +1,230 @@
+- **Processing Time:**
+  - Processing may take time depending on the number and size of files.
+  - The script uses multiprocessing to improve performance.
+
+- **Customizing Prompts:**
+  - You can adjust prompts in `data_processing.py` to change how metadata is generated.
+
+## License
+
+This project is dual-licensed under the MIT License and Apache 2.0 License. You may choose which license you prefer to use for this project.
+
+- See the [MIT License](LICENSE-MIT) for more details.
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)